knowledge-base/kb-changelocalization-at-runtime.md

```diff
@@ -0,0 +1,42 @@
+---
+title: PDFViewer Localization of The FindDialog During Runtime
+description: "How to change the localization at runtime."
+type: how-to
+page_title: How to change the localization of the FindDialog during runtime for WPF.
+slug: kb-changelocalization-at-runtime
```

Comment:
The slug should contain the name of the control in lowercase and each word should be separated with a hyphen "-".